fe/index.php

```diff
@@ -0,0 +1,132 @@
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <link href="https://fonts.googleapis.com/css?family=Anton&display=swap" rel="stylesheet">
+        <link rel="stylesheet" type="text/css" href="normalize.css">
+        <link rel="stylesheet" type="text/css" href="styles.css">
+        <title>Record Collection</title>
+    </head>
+    <body>
+        <header>
+            <h1>Record Collection</h1>
+        </header>
+    <main>
```

Comment:
you need to call your displayRecords function in this index.php file and then echo within the html (here) where you want your function to produce the html